File: app/src/main/java/com/example/pgidoctor/PatientDetails.kt
package com.example.pgidoctor

import java.io.Serializable

data class PatientDetails(
    val name: String = "",
    val fathername: String = "",
    val date: String = "",
    val crno: String = "",
    val mobile: String = "",
    val age: String = "",
    val gender: String = "",
    var profileImageUrl: String = "",
    var hospitalText: String="",
    var unitText: String = "",
    var id: String = "",
    var isstarred: String = "",
    var isimportant: String = "",
    var issevere: String = "",
    var isnearby: String = "",
    var email: String = ""
    ): Serializable

File: admin_dep/models/PatientDetails.js
class PatientDetails {
    constructor(age,crno,date,email,fathername,gender,hospitalText,id,isImportant,isSevere,isStarred,isNearby,mobile,name,profileImageUrl,unitText ) {
            this.id = id;
            this.email = email;
            this.name = name;
            this.mobile = mobile;
            this.age = age
            this.crno = crno
            this.date = date
            this.fathername = fathername
            this.gender = gender
            this.hospitalText = hospitalText
            this.id = id
            this.isImportant = isImportant
            this.isSevere = isSevere
            this.isStarred = isStarred
            this.isNearby = isNearby
            this.unitText = unitText
            this.profileImageUrl = profileImageUrl
    }
}

module.exports = PatientDetails;